TestingStaff/tstStaff.cs

```diff
@@ -261,31 +267,27 @@ public void TestFamilyNameFound()
             //Boolean variable to store the result of the search.
             Boolean Found = false;
 
-            // Boolean variable to record if the data is OK.
+            // Boolean variable to record if the data is OK. *CURRENTLY REDUNDANT *
```

Comment:
This could be annotated with `#warning this is redundant` instead.

Reply:
Removed. Was only added while I was testing different methods.